client/src/entrypoints/admin/telepath/widgets.js

```diff
@@ -363,7 +386,7 @@ class DraftailRichTextArea {
 
     const boundDraftail = new BoundDraftailWidget(
       input,
-      this.options,
+      { ...options, ...this.options },
```

Comment:
How about swapping these around, so that options passed to an individual widget instance via `render` can override the general ones?
```suggestion
      { ...this.options, ...options },
```

Reply:
Sounds like a good idea. Lets do it!